actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -219,22 +221,11 @@ workflows:
                 runfolder_name: "<% $.runfolder_name %>"
                 private_key: <% $.summary_host_key %>
               on-success:
-                - check_hosts_to_rsync_to
-
-            check_hosts_to_rsync_to:
-               action: core.http
-               input:
-                    url: <% $.hermes_base_url %>/<% $.flowcell_name %>/flowcell/analysishostinfo
-                    headers:
-                      USER: <% $.hermes_user %>
-                      X-XSRF-TOKEN: <% $.hermes_token %>
-               on-success:
-                    - rsync_to_milou: <% 'milou-b.uppmax.uu.se' in task(check_hosts_to_rsync_to).result.body.remote_hosts %>
-                    - make_irma_samplesheet_changes: <% 'irma.uppmax.uu.se' in task(check_hosts_to_rsync_to).result.body.remote_hosts %>
+                - make_irma_samplesheet_changes
```

Comment:
We have a ticket on removing this task since it is not needed anymore. I would suggest going directly to rsync_to_irma.

Reply:
Ok. I'll remove it. 👍